sdk/eventhub/tests.yml

```diff
@@ -13,8 +22,7 @@ stages:
         AZURE_STORAGE_DATA_LAKE_ENABLED_CONN_STR: $(python-eh-livetest-event-hub-storage-data-lake-enabled-conn-str)
         IOTHUB_CONNECTION_STR: $(python-eh-livetest-event-hub-iothub-connection-str)
         IOTHUB_DEVICE: $(python-eh-livetest-event-hub-iothub-device)
-        AZURE_CLIENT_ID: $(python-eh-livetest-event-hub-aad-client-id)
-        AZURE_TENANT_ID: $(python-eh-livetest-event-hub-aad-tenant-id)
-        AZURE_CLIENT_SECRET: $(python-eh-livetest-event-hub-aad-secret)
-        AZURE_SUBSCRIPTION_ID: $(python-eh-livetest-event-hub-subscription-id)
```

Comment:
hey @v-xuto , thanks for the PR, could you explain why is `AZURE_SUBSCRIPTION_ID` removed?

I ask it because when I kicked off a livetest running against your PR, tests are skipped in the running. Please check the run here: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1191265&view=logs&j=bcb4ccc7-07aa-5122-c376-b14b8283d586&t=3fb9b9b1-5d32-5c98-e32e-07d724d6cc51

also, in your test running: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1190562&view=results,
tests are also skipped.

usually, the tests are skipped because of missing environment variables.

Reply:
@yunhaoling Sorry, I have added 'AZURE_SUBSCRIPTION_ID' in tests.yml. I ran it in a pipeline, and it failed in public, Usgov, and China Cloud. For more details please check [here](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1192096&view=logs&j=84144e46-a9a2-5427-7a46-efb6820898b2&t=59e7c4c8-621a-5519-c391-a404e35f79b4). Error Message:  `The provided subscription identifier '$(python-eh-livetest-event-hub-subscription-id)' is malformed or invalid.`
@benbp How to set the environment variables about `AZURE_SUBSCRIPTION_ID`?

Reply:
@v-xuto I added a commit suggestion below. You can do it in the same it's done for the other variables you changed.